### PR TITLE
Linting all files in lib

### LINKED ISSUE #4: Lint all the files
## Description
Right now the file names and the code is not linted properly (or does not follow the convention naming etc.). Lint all the files.

## Multiple contributors
This issue can have multiple PRs but each PR should lint at least 3 files.

### PR DESCRIPTION
**Pull request by Starkl7**
## Issue
Fixes #04

<br>

## Description

Linted all files in lib, except Screens.
Mainly consisted of unused imports and incorrect camel case.

<br>

## Type of change

- Bug fix (non-breaking change which fixes an issue)

<br>

## Checklist:

-  My code follows the style guidelines of this project
-  The app runs properly
-  I have performed a self-review of my own code
-  I have formatted and commented my code, particularly in hard-to-understand areas
-  My changes generate no new warnings

### CODE REVIEW

**Comment by Saransh-cpp** [comment]
@Starkl7, now that #57 is done, can you merge in master? Let me know if you need any help!

**Comment by Starkl7** [comment]
@Saransh-cpp Resolved the conflict, looks good?

File: lib/Widgets.dart [hunk 1]
```diff
@@ -203,7 +206,7 @@ class ProductCard extends StatelessWidget {
 
   //==========================Product Image===========================
 
-  Widget _productImage(String picture) {
+/* Widget _productImage(String picture) {
```
Comment (Saransh-cpp) [inline lib/Widgets.dart:209]: Is this function not being used anywhere?

**Review by Saransh-cpp — CHANGES_REQUESTED** [review]
Thanks, @Starkl7! Looks greats! Some comments below (major ones are associated with merging issues) -

File: lib/Widgets.dart (hunk 1 above)
Comment (Starkl7) [reply lib/Widgets.dart:209]: No, this func was unused. Instead of deleting, I though it would be better to comment it out.

**Review by Saransh-cpp — APPROVED** [review]
Thanks, @Starkl7! Awesome work! The tests are finally passing. I'll turn this into an easy issue (doesn't look like a `good first issue` as you went through all the files)

Could you resolve the conflicts? I will run the app tomorrow once and then merge!

**Comment by Starkl7** [comment]
@Saransh-cpp Thanks for the consideration. 
I resolved all the issues, should be good to go now.

**Comment by Starkl7** [comment]
Some new issues have popped up in Register.dart. I fixed the ones I could, a few I could not.
Pls look into that @Saransh-cpp.

**Comment by Starkl7** [comment]
@Saransh-cpp Some old code had been reintroduced in Register.dart. Probably messed up somewhere 😬 
Anyways took care of it so it should be cool now.

File: lib/Screens/product_details.dart [hunk 2]
```diff
@@ -279,7 +257,7 @@ class _ProductDetailsState extends State<ProductDetails> {
                       appProvider.changeIsLoading();
                       return;
                     }
-                    appProvider.changeIsLoading();
+                    // appProvider.changeIsLoading();
```
Comment (Saransh-cpp) [inline lib/Screens/product_details.dart:260]: Why was this commented?

File: lib/Screens/product_details.dart [hunk 3]
```diff
@@ -301,7 +279,7 @@ class _ProductDetailsState extends State<ProductDetails> {
                 appProvider.changeIsLoading();
                 return;
               }
-              appProvider.changeIsLoading();
+              // appProvider.changeIsLoading();
```
Comment (Saransh-cpp) [inline lib/Screens/product_details.dart:282]: Why was this commented?

**Review by Saransh-cpp — COMMENTED** [review]
Awesome! Some minor comments below, also I'll put it up as moderate as this required a lot of work!

File: lib/Screens/product_details.dart (hunk 2 above)
Comment (Starkl7) [reply lib/Screens/product_details.dart:260]: That was dead code. 
Above it is an if...else... and there in both blocks, whether if or else, there is a return statement. Therefore, the program execution would never reach this line.

File: lib/Screens/product_details.dart (hunk 3 above)
Comment (Starkl7) [reply lib/Screens/product_details.dart:282]: Same reason as above... Dead (Unreachable) code

File: lib/Screens/product_details.dart (hunk 2 above)
Comment (Saransh-cpp) [reply lib/Screens/product_details.dart:260]: Oh oh, didn't see the code above, thanks!

**Comment by Saransh-cpp** [comment]
@Starkl7, you are only a part of SWoC right?

**Comment by Saransh-cpp** [comment]
@all-contributors please add @Starkl7 for code

**Comment by Starkl7** [comment]
@Saransh-cpp Thanks for putting it up as moderate. It felt great contributing to this project. 😄
Yes, I am participating in SWOC only.

**Pull request merged**